### ISSUE TITLE
License

### ISSUE THREAD

**Issue by Iiridayn** [gmorenz/pico-tts#1]
I've developed a fork which is a 50% rewrite, and I'm leaving the history behind to shrink download size. Is this repo licensed under a permissive license so I can do so? Or will I need to rewrite the other 50%? Unlicensed = full copyright.

**Comment by Iiridayn** [comment]
Btw, thank you for the work you've done on this already.

**Comment by gmorenz** [comment]
Oops, I missed this at the time. I'm sure this is years too late to be useful, but...

You can consider this me licensing my copyrights on this repo under CC0 (basically public domain).

I don't own any copyright on the actual pico-tts source code, it claims to be licensed under Apache-2 (license is included in the src dir and in the headers of files), but if used in a serious project you might want to figure out what is going on with the DMCA claims that made this repo necessary.

**Comment by Iiridayn** [comment]
Thank you. I was having trouble with the TTS crashing at the end and spitting out 1/10th of a second of static, so I rebuilt it just for personal use. I appreciate updating the copyright :). I figured the intention was probably sth to that effect.

Wrt the DMCA thing... wow. Probably an automated scanner/removal bot incorrectly banning use of Apache licensed code, etc, but I agree that's quite disturbing.
